jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/diagram/ZoomControlView.java

```diff
@@ -113,6 +113,22 @@ public HTMLElement getElement() {
         return buttons;
     }
 
+    public void disablePlusButton(boolean disabled) {
+        disableButton(scalePlus, disabled);
+    }
+
+    public void disableMinusButton(boolean disabled) {
+        disableButton(scaleMinus, disabled);
+    }
+
+    public void disableButton(HTMLAnchorElement anchorElement, boolean disabled) {
+        String buttonCssClass = "btn btn-default";
+        if (disabled) {
+            buttonCssClass += " disabled";
+        }
+        anchorElement.setAttribute("class", buttonCssClass);
```

Comment:
You can manipulate the class list directly using: `anchorElement.classList.add();` and  `anchorElement.classList.remove();`

Reply:
oks